<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BUT_play_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUT_stop_1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUT_play_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUT_play_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>play</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>BUT_play_1</zorder>
   <zorder>BUT_stop_1</zorder>
   <zorder>BUT_play_2</zorder>
   <zorder>BUT_play_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
